### PR TITLE
Spanish translation 

### PR DESCRIPTION
**Pull request by MRGABOZ**
I'm not adding external strings correctly let me know if I should fix something

### CODE REVIEW

File: shared/src/commonMain/kotlin/com/toasterofbread/spmp/resources/uilocalisation/localised/UILanguages.kt [hunk 1]
```diff
@@ -4,4 +4,5 @@ class UILanguages {
     val en: String = "en-GB"
     val ja: String = "ja-JP"
     val pl: String = "pl-PL"
-}
+    val es: String = "es-US
```
Comment (toasterofbread) [inline shared/src/commonMain/kotlin/com/toasterofbread/spmp/resources/uilocalisation/localised/UILanguages.kt:7]: Missing closing quote

**Review by toasterofbread — CHANGES_REQUESTED** [review]
Just one thing to fix, looks good other than that!

**Comment by MRGABOZ** [comment]
Ok perfect, I think I already corrected it  :)

**Comment by toasterofbread** [comment]
Thanks for the translation!

**Pull request merged**